.github/workflows/dev_builds.yml

```diff
@@ -150,17 +150,32 @@ jobs:
           AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID_DEV }}
           AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY_DEV }}
 
+      - name: Login to Docker Hub
+        run: |
+          docker login \
+            --username ${{ secrets.DOCKERHUB_LOGIN_DEV }} \
+            --password ${{ secrets.DOCKERHUB_PASSWORD_DEV }}
+
       - name: Download FIPS binary action artifact from build phase
         uses: actions/download-artifact@v4
         with:
           name: otelcol-sumo-fips-${{matrix.arch_os}}
 
-      - name: Build and push FIPS image to Open Source ECR
+      - name: Build and push FIPS images to Open Source ECR
```

Comment:
Looking at this stuff really makes me wish we used goreleaser instead :crying_cat_face:

Reply:
🙈